index.html

```diff
@@ -133,15 +133,15 @@
     <section id="sotd">
       <p>
         Since publication as Candidate Recommendation on <a href=
-        "http://www.w3.org/TR/2016/CR-presentation-api-20160714/">14 July
-        2016</a>, the Working Group updated most algorithms in the spec to fix
-        issues identified through testing and implementation feedback.
-        Interfaces defined in this document did not change, except
-        <code>PresentationConnectionClosedReason</code> and
-        <code>PresentationConnectionClosedEvent</code>, which were renamed to
-        improve consistency with the rest of the Web platform. The API was also
-        restricted to secure contexts. See the <a href=
-        "#changes-since-14-july-2016">list of changes</a> for details.
+        "https://www.w3.org/TR/2017/CR-presentation-api-20170601/">01 June
+        2017</a>, the Working Group updated the steps to construct a
+        <code>PresentationRequest</code> to ignore a URL with an unsupported
+        scheme, and dropped the definition of the <code>BinaryType</code> enum
+        in favor of the one defined in the HTML specification. Other interfaces
```

Comment:
You may want to mention the main changes from PR #436, which did update the normative text. Something like:

> placed further restrictions on how receiving browsing contexts are allowed to navigate themselves

Reply:
Done, thanks!